integration-libs/cdc/core/auth/services/user-authentication/cdc-user-register.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import {
+    AuthService,
+    Command,
+    CommandService,
+    UserActions,
+    WindowRef
+} from '@spartacus/core';
+import { User } from '@spartacus/user/account/root';
+import {
+    Title,
+    UserRegisterFacade,
+    UserSignUp
+} from '@spartacus/user/profile/root';
+import {
+    UserProfileConnector, UserProfileService
+} from 'feature-libs/user/profile/core';
+import { CdcJsService } from 'integration-libs/cdc/root';
+import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+@Injectable()
+export class CDCUserRegisterService implements UserRegisterFacade {
+  protected registerCommand: Command<{ user: UserSignUp }, User> =
+    this.command.create(
+      ({ user }) =>
+        new Observable<User>((userRegistered) => {
+          if (user.firstName && user.lastName && user.uid && user.password) {
+            this.cdcJSService.registerUserWithoutScreenSet(user);
+          }
+          this.store.dispatch(new UserActions.RegisterUserSuccess());
```

Comment:
RegisterUserSuccess is deprecated

Reply:
@dydom

I referred to the implementation in https://github.com/SAP/spartacus/blob/3dc96330abae9e74a582b657ab895b1e03c5019d/feature-libs/user/profile/core/facade/user-register.service.ts#L29

Any suggestions

Reply:
but your import is wrong I think, If you move mouse on this action it will inform you what to do and what you should use